packages/react-native-reanimated/Common/cpp/worklets/Tools/SingleInstanceChecker.h

```diff
@@ -52,10 +52,8 @@ SingleInstanceChecker<T>::SingleInstanceChecker() {
   std::string className =
       __cxxabiv1::__cxa_demangle(typeid(T).name(), nullptr, nullptr, &status);
 
-  // Only one instance should exist, but it is possible for two instances
-  // to co-exist during a reload.
   assertWithMessage(
-      instanceCount_ <= 1,
+      instanceCount_ < 1,
```

Comment:
When cleaning during invalidation stage, it shouldn't be possible for two instances to exist at the same time.

Reply:
Why not just `instanceCount_ == 0`?